data/upgrades/astromech.json

```diff
@@ -651,5 +651,129 @@
     "standard": true,
     "extended": true,
     "epic": true
+  },
+  {
+    "name": "R2-A3",
+    "limited": 1,
+    "xws": "r2a3-battleofyavin",
+    "sides": [
```

Comment:
Having sides on these abilities feels weird. Compatibility with other cards probably makes more sense than having the schema support two types of cards/abilities?

Reply:
Yes, keep sides to preserve compatibility.